### PR TITLE
"Getting Started with Sylius" guide

### PR DESCRIPTION
**Pull request by Zales0123**
| Q               | A
| --------------- | -----
| Branch?         | 1.3
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | 
| Related tickets | 
| License         | MIT

The idea of this documentation section is to show the simplest path from the installation of Sylius, through some configuration and customizations, to the deployment of the application to the server. It's, of course, the first version of such a guide, so there is a big space for improvements, but still, it should already be useful for the Sylius newcomers.

Chapters:

- [x] installation
- [x] basic-configuration
- [x] shipping-and-payment
- [x] first-product
- [x] shop-customizations
- [x] custom-business-logic
- [x] plugin-installation
- [x] deployment
- [x] summary

Zrzut ekranu 2019-05-14 o 16 17 54

### CODE REVIEW

File: docs/getting-started-with-sylius/custom-business-logic.rst [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    namespace App\Form\Type;
+
+    use Sylius\Bundle\MoneyBundle\Form\Type\MoneyType;
+    use Symfony\Component\Form\AbstractType;
+    use Symfony\Component\Form\Extension\Core\Type\NumberType;
+    use Symfony\Component\Form\FormBuilderInterface;
+
+    final class ParcelShippingCalculatorType extends AbstractType
+    {
+        public function buildForm(FormBuilderInterface $builder, array $options)
```
Comment (Tomanhez) [inline docs/getting-started-with-sylius/custom-business-logic.rst]: Missing type?

File: docs/getting-started-with-sylius/first-product.rst [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+shipping? Should it be tracked within the inventory system? On which channel will it be available? Should it have some taxes added during
+the checkout or not? Take a while to explore these options later, right now let's fill only the most critical data:
+
+.. image:: /_images/getting-started-with-sylius/product-creation.png
+
+Summary
+-------
+
+Great, the first stage is done! The whole checkout process is described in :doc:`this part of the documentation </book/orders/checkout>`,
+it's quite natural and logical, so it should not be a problem to understand it.
```
Comment (lchrusciel) [inline docs/getting-started-with-sylius/first-product.rst]: `so it should not be a problem to understand it.` I would be afraid of using this phrase, as if someone will not understand it, they may feel dumb
```suggestion

```

File: docs/getting-started-with-sylius/installation.rst [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    $ composer create-project sylius/sylius-standard MyFirstShop
+
+It will create a ``MyFirstShop`` directory with a brand new Sylius application inside.
+
+.. warning::
+
+    Beware! The next step includes the database setup. It will set your database credentials
+    (username, password, and database name) in the file with environment variables (``.env`` is the most basic one).
+
+To launch a Sylius application, inidial data has to be set up: an administrator account and base locale.
```
Comment (lchrusciel) [inline docs/getting-started-with-sylius/installation.rst]:
```suggestion
To launch a Sylius application initial data has to be set up: an administrator account and base locale.
```

File: docs/getting-started-with-sylius/plugin-installation.rst [hunk 4]
```diff
@@ -0,0 +1,40 @@
+Plugin installation
+===================
+
+Sylius is easy to customize to your business needs, but not all of the customizations you have to do on your own! Sylius
```
Comment (lchrusciel) [inline docs/getting-started-with-sylius/plugin-installation.rst]: Can we move the second Sylius to the new line?

File: docs/getting-started-with-sylius/basic-configuration.rst [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+----
+
+The last configuration step is creating a zone. They are used for various reasons, like shipping and taxing operations,
+and can consist of countries, provinces or other zones.
+
+.. image:: /_images/getting-started-with-sylius/zones-types.png
+    :scale: 55%
+    :align: center
+
+|
```
[the remark this answers is not in the document]
Comment (Zales0123) [reply docs/getting-started-with-sylius/basic-configuration.rst]: It's a blank line, without it, the text is much too close to the image

File: docs/getting-started-with-sylius/plugin-installation.rst (hunk 4 above)
Comment (Zales0123) [reply docs/getting-started-with-sylius/plugin-installation.rst]: #secondSylius :D 🎉

File: docs/index.rst [hunk 6]
```diff
@@ -23,6 +23,18 @@ Sylius Documentation
 
    For tips on using only some bundles of Sylius head to Bundles and Components docs.
 
+Getting Started with Sylius
+---------------------------
+
+Getting Started with Sylius
```
Comment (CoderMaggie) [inline docs/index.rst]: Well... I think we need sth more here :D

Comment (Zales0123) [reply docs/index.rst]: xD

**Comment by yaroslavche** [comment]
Hi.

In [Installation > Launching application](https://github.com/Zales0123/Sylius/blob/getting-started-with-sylius-guide/docs/getting-started-with-sylius/installation.rst#launching-application):

```bash
$ bin/console server:start
$ open http://127.0.0.1:8000/admin
```

I think `open http://127.0.0.1:8000/admin` must be not in code block. And maybe with link and small explanation why this link not always (but in most cases) http://127.0.0.1:8000.

As I can know `$` indicating that you have normal command prompt and `#` - superuser prompt.

```bash
$ su 
# 
```
I know, `#` symbol looks like a comment. And maybe this doc isn't a place for commands that requires a superuser access.

File: docs/getting-started-with-sylius/deployment.rst [hunk 7]
```diff
@@ -0,0 +1,139 @@
+Deployment
+==========
+
+Development usually takes most of the time in project implementation, but we should not forget about what's at the end of this process -
+application deployment into the server. We, in Sylius, believe, that it should be as easy and understandable as possible.
```
Comment (CoderMaggie) [inline docs/getting-started-with-sylius/deployment.rst]:
```suggestion
application deployment into the server. We believe, that it should be as easy and understandable as possible.
```

File: docs/getting-started-with-sylius/deployment.rst [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+There are many servers which you can choose for your store deployment, but in this chapter, we will use a `Platform.sh <https://platform.sh/>`_.
```
Comment (CoderMaggie) [inline docs/getting-started-with-sylius/deployment.rst]:
```suggestion
There are many servers which you can choose for your store deployment, but in this chapter, we will use `Platform.sh <https://platform.sh/>`_.
```

File: docs/getting-started-with-sylius/deployment.rst [hunk 9]
```diff
@@ -0,0 +1,139 @@
...
+
+As a result, you should see tons of logs and active status of the project when they pass:
+
+.. image:: /_images/getting-started-with-sylius/platform-sh-project-running.png
+    :scale: 55%
+    :align: center
+
+|
+
+You have also a URL provided, which you can visit to see is your shop working well. Of course, it does not have your configuration
```
Comment (CoderMaggie) [inline docs/getting-started-with-sylius/deployment.rst]:
```suggestion
You have also a URL provided, which you can visit to see if your shop working well. Of course, it does not have your configuration
```

File: docs/getting-started-with-sylius/shipping-and-payment.rst [hunk 10]
```diff
@@ -0,0 +1,45 @@
+Shipping & Payment
+==================
+
+The basic configuration is done. We can now proceed to letting potential customers to buy our merchandise.
```
Comment (CoderMaggie) [inline docs/getting-started-with-sylius/shipping-and-payment.rst]:
```suggestion
The basic configuration is done. We can now proceed to let potential customers buy our merchandise.
```

**Comment by pamil** [comment]
Thank you, Mateusz! :tada:

**Pull request merged**